### ISSUE TITLE
I find difficulty to put black keys in common piano arrangement.

### ISSUE THREAD

**Issue by albawiany** [puneet94/piano#1]
I like this app very much. Unfortunately however, I find difficulty to put black keys in common piano arrangement. Can we solve it?

**Comment by puneet94** [comment]
yeah we can solve it...I just didnt have the exact sounds(piano notes) with me when I was building this app.  The number of keys generated is also not similar to a piano. So we can work on it.

**Comment by albawiany** [comment]
the exact sounds(piano notes): http://theremin.music.uiowa.edu/MISpiano.html
